examples/react-alt/js/app.jsx

```diff
@@ -0,0 +1,187 @@
+/*jshint quotmark:false */
+/*jshint white:false */
+/*jshint trailing:false */
+/*jshint newcap:false */
+/*global React, Router*/
+var app = app || {};
+
+(function () {
+	'use strict';
+
+	app.ALL_TODOS = 'all';
+	app.ACTIVE_TODOS = 'active';
+	app.COMPLETED_TODOS = 'completed';
+	var TodoFooter = app.TodoFooter;
+	var TodoItem = app.TodoItem;
+	var TodoActions = app.todoActions;
+	var TodoStore = app.todoStore;
+
+	var ENTER_KEY = 13;
+
+	var TodoApp = React.createClass({
+		getInitialState: function () {
+			return TodoStore.getState();
+		},
+
+		componentDidMount: function () {
+			TodoStore.listen(this.onStoreChange);
+
+			var router = Router({
+				'/': function () {
+					TodoActions.show(app.ALL_TODOS);
+				},
+				'/active': function () {
+					TodoActions.show(app.ACTIVE_TODOS);
+				},
+				'/completed': function () {
+					TodoActions.show(app.COMPLETED_TODOS);
+				}
+			});
+
+			router.init('/');
+		},
+
+		componentDidUnmount: function () {
+			TodoStore.unlisten(this.onStoreChange);
+		},
+
+		onStoreChange: function (state) {
+			this.setState(state);
+		},
+
+		handleNewTodoKeyDown: function (event) {
+			if (event.keyCode !== ENTER_KEY) {
+				return;
+			}
+
+			event.preventDefault();
+
+			var val = React.findDOMNode(this.refs.newField).value.trim();
+
+			if (val) {
+				TodoActions.addTodo(val);
+				React.findDOMNode(this.refs.newField).value = '';
```

Comment:
I've updated the main React example for this. Would you mind adjusting the PR accordingly?